nano/node/messages.cpp

```diff
@@ -987,6 +987,11 @@ std::size_t nano::confirm_ack::size (std::size_t count)
 	return result;
 }
 
+std::string nano::confirm_ack::to_string () const
```

Comment:
This function only prints a fraction of the data that confirm_ack carries:
A confirm carries the following data:
* timestamp
* the account that signs the hashs
* signature
* a list of hashes

```
NetID: C, VerMaxUsingMin: 18/18/18, MsgType: 5(confirm_ack), Extensions: 1100
Account: ECAB1C2AFF0059E79FD7B9B33C2E2EA0FE825EA753D121CBBE3E0AB004B7F45C
         nano_3u7d5iohy14swyhxhgfm9iq4xa9yibhcgnyj697uwhicp14dhx4woik5e9ek
Signature: 1784890DE75A7480F03B5836CBFF1C5A282ED8D3121E5105A643499718B809E1C986C823C823594D4B6A49DDDC4DA7359FD82143E79F145647C428CB5B7AB003
Sequence: 18446744073709551615(0xffffffffffffffff) [final vote]
Valid signature
Hashes:
   991CF190094C00F0B68E2E5F75F6BEE95A2E0BD93CEAA4A6734DB9F19B728948
```

Reply:
Also, it should print the header.